### ISSUE TITLE
[PRE REVIEW]: Statsomat/CFA: A Shiny app for automated confirmatory factor analysis

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#3404]
**Submitting author:** @reyar (<a href="http://orcid.org/0000-0001-8904-1631">Denise Welsch</a>)
**Repository:** <a href="https://github.com/Statsomat/CFA" target ="_blank">https://github.com/Statsomat/CFA</a>
**Version:** v1.0
**Editor:** Pending
**Reviewer:** Pending
**Managing EiC:** Kyle Niemeyer

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/dc37a1bc305141b1a88aa9f5e933ebbd/status.svg)](https://joss.theoj.org/papers/dc37a1bc305141b1a88aa9f5e933ebbd)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/dc37a1bc305141b1a88aa9f5e933ebbd"><img src="https://joss.theoj.org/papers/dc37a1bc305141b1a88aa9f5e933ebbd/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/dc37a1bc305141b1a88aa9f5e933ebbd/status.svg)](https://joss.theoj.org/papers/dc37a1bc305141b1a88aa9f5e933ebbd)
```

**Author instructions**

Thanks for submitting your paper to JOSS @reyar. **Currently, there isn't an JOSS editor assigned** to your paper.

@reyar if you have any suggestions for potential reviewers then please mention them here in this thread (without tagging them with an @). In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission (please start at the bottom of the list).

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.88  T=0.03 s (862.8 files/s, 129930.5 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Rmd                              5            429            515           1273
R                                5            245             53            644
HTML                             9              8              0            117
TeX                              2             22              0            108
Markdown                         2             26              0             90
JavaScript                       1             19              1             64
-------------------------------------------------------------------------------
SUM:                            24            749            569           2296
-------------------------------------------------------------------------------


Statistical information for the repository 'b3307a49034bbed889658821' was
gathered on 2021/06/25.
The following historical commit information, by author, was found:

Author                     Commits    Insertions      Deletions    % of changes
Denise Welsch                    1            84              0          100.00

Below are the number of rows from each author that have survived and are still
intact in the current revision:

Author                     Rows      Stability          Age       % in comments
Denise Welsch                84          100.0          0.0                1.19
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- None

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03404/joss.03404/10.21105.joss.03404.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03404/joss.03404/10.21105.joss.03404.pdf) :page_facing_up: :point_left:

**Comment by kyleniemeyer** [comment]
Hi @reyar, thanks for your interest in JOSS. Due to the relatively small size of your submission, the editorial board is going to review your submission to see if it meets our requirements for [substantial scholarly effort](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort).

**Comment by kyleniemeyer** [comment]
@whedon query scope

**Comment by whedon** [comment]
Submission flagged for editorial review.

**Comment by reyar** [comment]
Hi @kyleniemeyer, thanks. Should I make a suggestion for potential reviewers or should I wait to see if my submission meets the requirements for substantial scholarly effort?

**Comment by reyar** [comment]
@kyleniemeyer Maybe this information plays a role in your decision: The software report from above does not count the work from the repository https://github.com/Statsomat/CFA-Testing which was used for the testing and which is also mentioned in the README file. There are non-technical reasons for the existance of 2 repositories instead of one.

**Comment by reyar** [comment]
Hi @kyleniemeyer I hope I'm not too importunate :) But JOSS already published a Shiny application which is very similar to mine: https://github.com/PsyChiLin/EFAshiny.

**Comment by arfon** [comment]
> Hi @kyleniemeyer I hope I'm not too importunate :) But JOSS already published a Shiny application which is very similar to mine: https://github.com/PsyChiLin/EFAshiny.

Thanks for pointing out this other submission. I would note that the https://github.com/PsyChiLin/EFAshiny submission was published ~3 years ago in JOSS and our scope has evolved somewhat since then. 

This isn't to say your submission will be deemed out of scope (the wider editorial team will make a decision on this), rather that previous submissions should not be relied upon as evidence that a similar submission today would still in scope today.

**Comment by reyar** [comment]
Thanks for the reply.

I still want to mention that my app Statsomat/CFA offers something which does not exist yet: Not only a bunch of numerical and graphical outputs but also a text-interpretation in plain English. This is the final scope of my project https://statsomat.com.

 Since I have two submissions with the same question of substantial scholarly effort and at least one more is planned for submission, I would be very happy if you could consider them also as pieces of the project https://statsomat.com. 

I could also put them together but I am not so sure if this is ideal.

Thanks.

Dr. Denise Welsch
Rey Analytical Research 
https://reyar.de
https://statsomat.com 

Am 27.06.2021 um 10:42 schrieb Arfon Smith ***@***.***>:

﻿
Hi @kyleniemeyer I hope I'm not too importunate :) But JOSS already published a Shiny application which is very similar to mine: https://github.com/PsyChiLin/EFAshiny.

Thanks for pointing out this other submission. I would note that the https://github.com/PsyChiLin/EFAshiny submission was published ~3 years ago in JOSS and our scope has evolved somewhat since then.

This isn't to say your submission will be deemed out of scope (the wider editorial team will make a decision on this), rather that previous submissions should not be relied upon as evidence that a similar submission today would still in scope today.

**Comment by reyar** [comment]
Hi @arfon and @kyleniemeyer I was wondering if you had the time to review my submissions to JOSS? 

Thanks.

**Comment by arfon** [comment]
@reyar - many thanks for your submission to JOSS. Unfortunately packages such as this that provide a limited GUI wrapper around another tool are generally classified as out of scope for us. This comment also applies to your other submission #3422.

> Since I have two submissions with the same question of substantial scholarly effort and at least one more is planned for submission, I would be very happy if you could consider them also as pieces of the project https://statsomat.com.

We've discussed this as an editorial team and the general consensus is that a combined submission would also not be allowed for JOSS (rationale being that many out of scope tools doesn't make an in-scope one).

One possible alternative to JOSS is to follow [GitHub's guide](https://guides.github.com/activities/citable-code/) on how to create a permanent archive and DOI for your software. This DOI can then be used by others to cite your work.

**Comment by arfon** [comment]
@whedon reject

**Comment by whedon** [comment]
Paper rejected.
